src/constants.js

```diff
@@ -55,3 +55,6 @@ export const SAMPLER_TYPE_REMOTE = 'remote';
 // The value of the header is recorded as the tag on the root span, so that the
 // trace can be found in the UI using this value as a correlation ID.
 export const JAEGER_DEBUG_HEADER = 'jaeger-debug-id';
+
+// TRACER_BAGGAGE_HEADER_PREFIX is the default prefix used for saving baggage to a carrier.
+export const TRACER_BAGGAGE_HEADER_PREFIX = 'uberctx-';
```

Comment:
this is uber specific, is there something that can be done to make it generic for OSS purposes?